.github/ISSUE_TEMPLATE/documentation.yaml

```
@@ -0,0 +1,40 @@
+name: Documentation
+description: Write a report about the issue
```

```suggestion
description: A request to update or improve Euphony documentation.
```

how about this? more than clear.